minutes/2022-06-13.md

```diff
@@ -0,0 +1,61 @@
+# Meeting Minutes for February 25, 2021
+
+## Attendees
+
+@fviernau, @mbochtler, @mnonnenmacher, @sschuberth, @tsteenbe
+
+## Extending the TSC
+
+On proposal of @mnonnenmacher the TSC members unanimously vote for adding Marcel Bochtler (@MarcelBochtler) as a new TSC
+member and Marcel accepts the vote.
+
+## Update from participants on future plans for ORT
+
+### Bosch
+
+We will soon start working on an ORT server backend. The main requirements are:
+- Horizontal scalability to run tasks like the analyzer and scanner in parallel.
+- Integration with enterprise authentication.
+- API to trigger jobs and request results.
+- Backed by database, likely PostgreSQL.
+- Should be cloud agnostic.
+- Code will be open source as soon as possible.
+
+Sebastian suggests that the CLI could later on be adapted to work with a local server in the background.
+
+### EPAM
+
+Upcoming topics for Frank:
+- GoMod support improvements
+- publish orth bash script repository
+- SPDX support - update to 2.3 and later 3.0
+- Package configurations: Helper command to generate path excludes for common directories like "docs"
+
+Thomas focuses on:
+- Make ORT more consumable, biggest feedback from users is difficult usability and onramp.
+- Present ORT in the LF ospology live meetups: https://github.com/todogroup/ospology/blob/main/ospology-live/framework.md
+- Find a maintenance solution for the web app reporter.
```

Comment:
Find a maintenance solution for the web app reporter. -> Find a additional maintainer for the web app reporter.